chips/cc26x2/src/aon.rs

```diff
@@ -136,16 +183,25 @@ impl Aon {
     }
 
     pub fn lfclk_enable(&self, enable: bool) {
-        let regs = AON_IOC_BASE;
+        let regs = &*self.ioc_regs;
         if enable {
             regs.ioc_clk32k_ctl.write(IocClk::EN::SET);
```

Comment:
can't you just just change this to self.ioc.regs.ioc_clk32k_ctl? This dereferenced pointer business (is that a raw pointer?) is kind of funky